docs/service-guide/update-a-onepassword-secret.rst

```diff
@@ -0,0 +1,30 @@
+#####################################################
+Updating a secret stored in 1Password and VaultSecret
+#####################################################
+
+Secrets that are stored in 1Password are synchronized into Vault using the `installer/generate_secrets.py <https://github.com/lsst-sqre/phalanx/blob/master/installer/generate_secrets.py>`__ script.
+Once they are in Vault, they are accessible to the Vault Secrets Operator, which responds to creation of any ``VaultSecret`` resources in Kubernetes by grabbing the current value of the secret data in Vault.
+
+The Vault Secrets Operator doesn't independently detect if there are changes to data in Vault, though.
+It only provides a snapshot of the data in Vault at the time the ``VaultSecret`` was created.
+
+So, if you want to make any changes to a ``VaultSecret``'s data, you'll need to:
+
+1. Make the changes in 1Password
+2. Run the `installer/update_secrets.sh <https://github.com/lsst-sqre/phalanx/blob/master/installer/update_secrets.sh>`__ script, as described in :doc:`/service-guide/add-a-onepassword-secret`.
+3. Delete the ``VaultSecret`` in the ArgoCD UI.
+4. Sync your application in ArgoCD, recreating the ``VaultSecret``, and triggering the Vault Secrets Operator to incorporate your changes.
+
+
+Deleting the ``VaultSecret``
```

Comment:
I think you can drop this section entirely; I don't think there's any reason why one would want to do this outside of deleting the whole application or syncing a new version.